src/App.tsx

```diff
@@ -43,11 +17,26 @@ function App() {
       "That is another one"
     )
   ]);
+  const [selectedIndex, setSelectedIndex] = useState(0);
 
-  const searchCtl = new SearchController();
   // const [searchResults, setSearchResults] = useState<Action[]>([]);
   // const [searchVisible, setSearchVisible] = useState(false);
 
+  const [yesNoVisible, setYesNoVisible] = useState(false);
```

Comment:
możesz zdefiniować sobie dostępne warianty dialogów w enumie i mieć jeden useState, w którym będziesz przechowywał informacje o obecnie wybranym wariancie

Reply:
ok, to się będzie lepiej skalować na kolejne dialogi :) podmieniam